server/src/domain/system-config/dto/system-config-oauth.dto.ts

```diff
@@ -47,4 +52,9 @@ export class SystemConfigOAuthDto {
 
   @IsString()
   storageLabelClaim!: string;
+
+  @ValidateIf(isEnabled)
+  @IsNotEmpty()
```

Comment:
I don't think we ever need to allow this being unspecified. It should just always be validated and always have a value.